### PR TITLE
initial take at breaking up into tower_config role

### PR DESCRIPTION
**Pull request by scollier**
The goal of this PR was to break up the monolithic tower_config.yml file into an Ansible role to make it more modular and follow best practices.  

The actual execution of any code in the repo does not change as a result of this refactor.

I tested with both tower_config: set to true and false.

### CODE REVIEW

File: aws_lab_launch_roles.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
```
[the remark this answers is not in the document]
Comment (scollier) [reply aws_lab_launch_roles.yml]: removed.

File: roles/tower_config/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+Role Name
```
[the remark this answers is not in the document]
Comment (scollier) [reply roles/tower_config/README.md]: removed

File: roles/tower_config/meta/main.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
+galaxy_info:
```
[the remark this answers is not in the document]
Comment (scollier) [reply roles/tower_config/meta/main.yml]: removed

File: aws_lab_launch.yml [hunk 4]
```diff
@@ -76,5 +76,12 @@
         record: "tower-{{ hostvars[inventory_hostname].student_id }}.{{ domain_name }}"
         value: "{{ hostvars[inventory_hostname].public_ip }}"
 
-- include: tower_config.yml
-  when: tower_config
+- name: Include tower_config role
+  hosts: tower_instances
```
[the remark this answers is not in the document]
Comment (scollier) [reply aws_lab_launch.yml]: added

**Comment by vvaldez** [comment]
Nice job!

**Pull request merged**